doc/README.md

```diff
@@ -1,5 +1,24 @@
 # Composite Setup Guide
 
+## Tools nessesary for Composite
+```
+sudo apt-get -y install bc  
+sudo apt-get -y install gcc-multilib  
+sudo apt-get -y install binutils-dev  
+sudo apt-get -y install qemu-kvm  
```

Comment:
I always install `qemu` vs `qemu-kvm` here. Does it matter?

Reply:
https://www.quora.com/Virtualization-What-is-the-difference-between-KVM-and-QEMU